src/components/molecules/Main/Main.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect} from 'react';
+import {CustomButton} from '../../atoms';
+import {View} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
+import {ROUTES, RootStackParamList} from '../../../static/types/routeTypes';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {globalStyles} from '../../../utils/globalStyles';
+
+type TMainProps = {
+  setIndex: React.Dispatch<React.SetStateAction<number>>;
+};
+export const Main = ({setIndex}: TMainProps) => {
+  const navigation = useNavigation<StackNavigationProp<RootStackParamList>>();
+
+  useEffect(() => {
+    if (navigation.isFocused()) {
+      setIndex(0);
+    }
+  }, [navigation.isFocused()]);
+  return (
+    <View
+      style={[
+        globalStyles.navigationWrapper,
+        {
+          alignItems: 'center',
```

Comment:
Nie optymalnie, w stylach to powinno być, nie ma powodu do tworzenia tutaj tablicy i obiektu

Reply:
tylko  że nie mam pewności co do tego czy każdy component który będzie używany do nawigacji będzie potrzebował alignItems: 'center'

Reply:
To zrób osobny styl po prostu